From 3f47e9588c8acf5c4cc5079e746bc05e7d5c69ec Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Tue, 24 Mar 2015 16:02:34 +0800
Subject: [PATCH] MLK-11065-8 bcmdhd: fixed build errors with 3.9.2 toolchain

  CC [M]  drivers/net/wireless/bcmdhd/dhd_ip.o
  LD [M]  fs/udf/udf.o
drivers/net/wireless/bcmdhd/dhd_common.c:108:22: error: macro "__DATE__" might prevent reproducible builds [-Werror=date-time]
  DHD_COMPILED " on " __DATE__ " at " __TIME__;
                      ^
drivers/net/wireless/bcmdhd/dhd_common.c:108:38: error: macro "__TIME__" might prevent reproducible builds [-Werror=date-time]
  DHD_COMPILED " on " __DATE__ " at " __TIME__;

Drop debug version with build date/time.
The kernel already has this information, and individual drivers
shouldn't duplicate that.  This also eliminates the use of __DATE__ and
__TIME__, which make the build non-deterministic.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 drivers/net/wireless/bcmdhd/dhd_common.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/net/wireless/bcmdhd/dhd_common.c b/drivers/net/wireless/bcmdhd/dhd_common.c
index c57874c..31ccb49 100644
--- a/drivers/net/wireless/bcmdhd/dhd_common.c
+++ b/drivers/net/wireless/bcmdhd/dhd_common.c
@@ -103,12 +103,7 @@ bool ap_fw_loaded = FALSE;
 #define DHD_COMPILED "\nCompiled in " SRCBASE
 #endif /* DHD_DEBUG */
 
-#if defined(DHD_DEBUG)
-const char dhd_version[] = "Dongle Host Driver, version " EPI_VERSION_STR
-	DHD_COMPILED " on " __DATE__ " at " __TIME__;
-#else
 const char dhd_version[] = "\nDongle Host Driver, version " EPI_VERSION_STR "\nCompiled from ";
-#endif 
 
 void dhd_set_timer(void *bus, uint wdtick);
 
-- 
1.8.0

